src/containers/Header.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Link, Router, IndexRoute, Route, browserHistory } from 'react-router';
+
+
+export default class Header extends Component {
+
+  handleLocation() {
+    const geolocation = require('geolocation');
+    const geo = navigator.geolocation;
+    let lat;
+    let lon;
+    navigator.geolocation.getCurrentPosition(function(position) {
```

Comment:
Interesting, I will look at taking those lat and long coordinates and passing them directly into the larger Weather API call, all as one function.  Thanks Steve!